### PR TITLE
Hide data latency graph

### LINKED ISSUE #2492: Hide Data Latency Graph Temporarily
### Describe the bug

The Data Latency graph is still showing same issue so we might as well hide it for now until we have a fix

### Steps to reproduce

1. Go to Log Analysis Overview
2. See issue in Data Latency chart (shows Y axis in billions)

### Expected behavior

Should be in a more reasonable time frame

### PR DESCRIPTION
**Pull request by kouknick**
## Description

Temporarily hide the Data Latency graph until we have a fix for the invalid data.

Closes #2492 

### Screenshots:

image

### Checklist:

- [x] Make sure you give an high-level overview of the problem & the solution
- [x] Make sure the related issue is referenced
- [x] Make sure screenshots are added if the UI is changed in any way
- [x] Make sure all of the significant logic is covered by tests
- [x] Make sure your changes are accessible (addition of aria-attributes & correct HTML semantics)
- [x] Make sure you reference the related design document if it exists
- [x] Make sure telemetry is added (analytics & errors)

### CODE REVIEW

File: web/src/pages/LogAnalysisOverview/LogAnalysisOverview.test.tsx [hunk 1]
```diff
@@ -121,7 +121,8 @@ describe('Log Analysis Overview', () => {
     ];
   });
 
-  it('should render 2 canvas, click on tab button and render latency chart', async () => {
+  // Skip this test until we re-enable data latency graph
+  it.skip('should render 2 canvas, click on tab button and render latency chart', async () => {
```
Comment (alexmylonas) [inline web/src/pages/LogAnalysisOverview/LogAnalysisOverview.test.tsx:125]: perfecto!

**Review by alexmylonas — APPROVED** [review]
LGTM! Optionally, you can also remove the parameter for fetching those datapoints, since we are not using them

**Comment by kouknick** [comment]
> LGTM! Optionally, you can also remove the parameter for fetching those datapoints, since we are not using them

Good catch! Removed it!

**Review by alexmylonas — COMMENTED** [review]
LGTM! 🚢  it

File: api/graphql/schema.graphql [hunk 2]
```diff
@@ -552,7 +552,9 @@ type SingleValue {
 type LogAnalysisMetricsResponse {
   eventsProcessed: LongSeriesData!
   alertsBySeverity: LongSeriesData!
+  """
```
Comment (vorillaz) [inline api/graphql/schema.graphql:555]: I think that was a typo.

File: web/src/pages/LogAnalysisOverview/LogAnalysisOverview.test.tsx [hunk 3]
```diff
@@ -93,7 +93,7 @@ describe('Log Analysis Overview', () => {
               'eventsProcessed',
               'totalAlertsDelta',
               'alertsBySeverity',
-              'eventsLatency',
+              // 'eventsLatency',
```
[the remark this answers is not in the document]
Comment (vorillaz) [reply web/src/pages/LogAnalysisOverview/LogAnalysisOverview.test.tsx:96]: Scratch that. I got it now. Please add a `//TODO` comment with notes for upcoming maintainers. If possible reference this PR along with the ticket in all places it should be addressed.

[the remark this answers is not in the document]
Comment (kouknick) [reply web/src/pages/LogAnalysisOverview/LogAnalysisOverview.test.tsx:97]: Done! Both PR and ticket are addressed in every file changed!

**Pull request merged**